### PR TITLE
rename comprehension to evidence everywhere user-facing

### PR DESCRIPTION
**Pull request by emilia-friedberg**
## WHAT
Rename "Comprehension" to "Evidence" everywhere user-facing, in the public urls, and in the code where it would have been more confusing to leave as is.

## WHY
We are moving forward with Quill Evidence as the name for now.

## HOW
Changed the name and key locally, then did a global search to find places that needed to be updated either because they explicitly used the wrong name or because they relied on the old value.

### Screenshots
(If applicable. Also, please censor any sensitive data)

### Notion Card Links
https://www.notion.so/quill/Rename-Comprehension-to-Quill-Evidence-ab7ce4bdfeb741aa9cff5ada100b83a3

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? | YES
Have you deployed to Staging? | YES
Self-Review: Have you done an initial self-review of the code below on Github? | YES
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | YES

### CODE REVIEW

**Review by eadams17 — APPROVED** [review]
the evidence is clear, this is good to go ✨

File: services/QuillLMS/app/assets/stylesheets/shared/icons.scss [hunk 1]
```diff
@@ -166,7 +166,7 @@
   background-position: center;
   height: 40px;
 }
-.icon-comprehension-green-no-border, .icon-9-green-no-border {
+.icon-evidence-green-no-border, .icon-9-green-no-border {
```
Comment (anathomical) [inline services/QuillLMS/app/assets/stylesheets/shared/icons.scss:169]: Appreciate clean-up even in things like class-names here.

File: services/QuillLMS/app/controllers/application_controller.rb [hunk 2]
```diff
@@ -8,6 +8,7 @@ class ApplicationController < ActionController::Base
   GOOGLE_OR_CLEVER_JUST_SET = :google_or_clever_just_set
   KEEP_ME_SIGNED_IN = :keep_me_signed_in
   COMPREHENSION = 'comprehension'
+  EVIDENCE = 'evidence'
```
Comment (anathomical) [inline services/QuillLMS/app/controllers/application_controller.rb:11]: Do we need to remove line 10 here with the `COMPREHENSION` definition?

File: services/QuillLMS/app/controllers/pages_controller.rb [hunk 3]
```diff
@@ -467,7 +467,7 @@ def backpack
     @style_file = 'staff'
   end
 
-  def comprehension
+  def evidence
     allow_iframe
     @style_file = ApplicationController::COMPREHENSION
```
Comment (anathomical) [inline services/QuillLMS/app/controllers/pages_controller.rb:472]: Does this need to be updated?  It looks like the answer is probably "no" (same thing on line 524 below), but I wanted to double check.

**Review by anathomical — APPROVED** [review]
Looks good to me.  I assume we also need to actually update the values in the `ActivityClassification` table.  Should we include a rake task for that here, or just update them by hand around the time this deploys?

File: services/QuillLMS/app/controllers/application_controller.rb (hunk 2 above)
Comment (emilia-friedberg) [reply services/QuillLMS/app/controllers/application_controller.rb:11]: Good question - no, this has to stay until we change the webpack definition for the file names, which I think we are going to do as part of a code-name change at a later date.

File: services/QuillLMS/app/controllers/pages_controller.rb (hunk 3 above)
Comment (emilia-friedberg) [reply services/QuillLMS/app/controllers/pages_controller.rb:472]: Same as above!

**Pull request merged**